src/index.js

```diff
@@ -58,16 +59,30 @@ const unvisit = (input, schema, getDenormalizedEntity) => {
     return method(schema, input, unvisit, getDenormalizedEntity);
   }
 
+  if (input === undefined || input === null) {
```

Comment:
I think there was a PR for this and it was randomly closed... #229. Makes sense to fix. We don't ever run across that case, but I could see it being possible.